packages/website/pages/account.js

```diff
@@ -40,6 +94,9 @@ export default function Account({ user }) {
     return () => clearTimeout(timer)
   }, [copied])
 
+  const isLoaded = !isLoading && !isFetching
+
+  const hasUsedTokensToUploadBefore = tokens.some(t => Object.entries(t.uploads?.data || {}).length > 0)
```

Comment:
`t.uploads?.data` will always be an array
```suggestion
  const hasUsedTokensToUploadBefore = tokens.some(t => t.uploads.data.length > 0)
```